flair/trainers/sequence_tagger_trainer.py

```diff
@@ -247,6 +250,16 @@ def evaluate(self, evaluation: List[Sentence], out_path=None, evaluation_method:
             score = metric.f_score()
             return score, metric
 
+    def log_metric(self, metric: Metric, dataset_name: str):
+        log.info("{0:<4}: f-score {1:.4f} - acc {2:.4f} - tp {3} - fp {4} - fn {5} - tn {6}".format(
+            dataset_name, metric.f_score(), metric.accuracy(), metric.get_tp(), metric.get_fp(),
+            metric.get_fn(), metric.get_tn()))
+        for cls in metric.get_classes():
+            log.info("{0:<4}: f-score {1:.4f} - acc {2:.4f} - tp {3} - fp {4} - fn {5} - tn {6}".format(
```

Comment:
As `metric.get_classes()` returns a list of all classes containing `None` this currently fails when training a sequence tagger due to 
```
TypeError: unsupported format string passed to NoneType.__format__
```
I would remove the `None` class from `metric.get_classes()`, so you don't log the results twice. Or you can use `dataset_name` in case of `cls is None` instead of using `cls` as name.

Reply:
I removed the `None` class from get_classes(), as it seems more consistent! Thanks!